share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
  |  FreeBSD           |      |                    |                    |
  |   13.2             |      |                    |                    |
  |                    |      |                    |                    |
-FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -current
+ *--FreeBSD           |      |                    |                    |
+ |   14.0 prerelease  |      |                    |                    |
+ |                    |      |                    |                    |
+FreeBSD 15.0-CURRENT  |  NetBSD-current  OpenBSD-current  DragonFly-DEVELOPMENT
  |                    |      |                    |                    |
  v                    v      v                    v                    v
 
 Time
 ----------------
 
-Time tolerance +/- 6 months, depending on which book/article you read; if it
-was the announcement in Usenet or if it was available as tape.
+Time tolerance on 4.3BSD and earlier releases can be as large as six months due
+to the uncertainty of historical records; the early release to special groups;
+and the then slowness of tape duplication. Articles and books from the period
+have different dates, since they relied on different sources: newest file in
+the distribution vs public announcement vs first available tape.
 
-[44B] McKusick, Marshall Kirk, Keith Bostic, Michael J Karels,
-	and John Quarterman. The Design and Implementation of
-	the 4.4BSD Operating System.
-[APL] Apple website [https://www.apple.com/macosx/]
+[44B]  McKusick, Marshall Kirk, Keith Bostic, Michael J Karels, and John
```

Comment:
this picked up an extra space, why?
Also it moved things around, why? The old way seemed better to read visually.